cmd/cloud/organization.go

```diff
@@ -249,3 +255,184 @@ func userUpdate(cmd *cobra.Command, args []string, out io.Writer) error {
+		Short:   "Update an Astro team",
+		Long:    "Update an Astro team",
+		Args:    cobra.MaximumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return teamUpdate(cmd, out, args)
+		},
+	}
+	cmd.Flags().StringVarP(&teamName, "name", "n", "", "The Team's name. If the name contains a space, specify the entire name within quotes \"\" ")
+	cmd.Flags().StringVarP(&teamDescription, "description", "d", "", "Description of the Team. If the description contains a space, specify the entire team description in quotes \"\"")
+	return cmd
+}
+
+func teamUpdate(cmd *cobra.Command, out io.Writer, args []string) error {
+	cmd.SilenceUsage = true
+
+	id := ""
+
+	if len(args) == 1 {
+		id = args[0]
+	}
+
+	return team.UpdateTeam(id, teamName, teamDescription, out, astroCoreClient)
+}
+
+func newTeamCreateCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "create",
+		Aliases: []string{"cr"},
+		Short:   "Create an Astro Team",
+		Long:    "Create an Astro Team",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return teamCreate(cmd, out)
+		},
+	}
+	cmd.Flags().StringVarP(&teamName, "name", "n", "", "The Team's name. If the name contains a space, specify the entire team within quotes \"\" ")
+	cmd.Flags().StringVarP(&teamDescription, "description", "d", "", "Description of the Team. If the description contains a space, specify the entire team in quotes \"\"")
+	return cmd
+}
+
+func teamCreate(cmd *cobra.Command, out io.Writer) error {
+	cmd.SilenceUsage = true
+	return team.CreateTeam(teamName, teamDescription, out, astroCoreClient)
+}
+
+func newTeamDeleteCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "delete [team_id]",
+		Aliases: []string{"de"},
+		Short:   "Delete an Astro Team",
+		Long:    "Delete an Astro Team",
+		Args:    cobra.MaximumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return teamDelete(cmd, out, args)
+		},
+	}
+	return cmd
+}
+
+func teamDelete(cmd *cobra.Command, out io.Writer, args []string) error {
+	cmd.SilenceUsage = true
+
+	id := ""
+
+	if len(args) == 1 {
+		id = args[0]
+	}
+
+	return team.Delete(id, out, astroCoreClient)
+}
+
+func newOrganizationTeamUserRootCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "user",
+		Short: "Manage users in your Astro Team",
+		Long:  "Manage users in your Astro Team.",
+	}
+	cmd.SetOut(out)
+	cmd.AddCommand(
+		newTeamRemoveUserCmd(out),
+		newTeamAddUserCmd(out),
+		newTeamListUsersCmd(out),
+	)
+	return cmd
+}
+
+func newTeamRemoveUserCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "remove",
+		Short: "Remove a user from an Astro Team",
+		Long:  "Remove a user from an Astro Team",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return removeTeamUser(cmd, out)
+		},
+	}
+	cmd.Flags().StringVarP(&teamID, "team_id", "t", "", "The Team's id \"\" ")
+	cmd.Flags().StringVarP(&userID, "user_id", "u", "", "The User's id \"\"")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&teamID, "team-id", "t", "", "The Team's id \"\" ")
	cmd.Flags().StringVarP(&userID, "user-id", "u", "", "The User's id \"\"")
```
nit: we use hyphens over underscore for all our flags.
Also maybe we can use a better flag description, maybe Jake Witz can help out to get the flag descriptions right.

Reply:
done